puppet/zulip_ops/manifests/base.pp

```diff
@@ -19,9 +19,8 @@
                         # Needed for zulip-ec2-configure-network-interfaces
                         "python3-six",
                         "python-six",
-                          # This one is needed for postgres as well
                         # "python3-boto", # missing on trusty
```

Comment:
I think that change (switching to `pip` for installing Python library dependencies on the system) would be reasonable where needed -- it's what we do for `wal-e` itself. The major downside is that when we stick to the versions in the distro release, we have some assurance that they're probably reasonably compatible, whereas if we switch to `pip` we take on the need to maintain a suite of pinned versions, just like we already do in our `requirements/` files.

But we should upgrade the version of `wal-e` we use anyway -- it hasn't been bumped since 2013 -- and I think that will solve the problem for its dependencies.